addons/base_import/static/src/js/import_action.js

```diff
@@ -110,6 +110,9 @@ var DataImport = Widget.extend(ControlPanelMixin, {
         this.session = session;
         action.display_name = _t('Import a File'); // Displayed in the breadcrumbs
         this.do_not_change_match = false;
+        // The reload feature is only supported on POSIX-like OS using Chrome
+        this.show_reload = (navigator.userAgent.indexOf(/Chrome/i) !== -1
+                            && navigator.userAgent.indexOf(/Windows/i) === -1);
```

Comment:
At that point I would rather we removed the feature entirely, and e.g. emptied the file input after getting its data in order to force the user to re-load the file, because limiting the feature to Chrome on Linux doesn't sound very useful.

Reply:
nginx closes the connection after one minute when data are missing https://bin.drlazor.be/entressong , the request never reaches the proxied server.